core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphQLUtils.kt

```diff
@@ -165,4 +170,273 @@ object GraphQLUtils {
         }
     }
 
+
+    data class GraphInfo(
+            var fields: MutableSet<String> = mutableSetOf(),
+            var edges: MutableSet<String> = mutableSetOf()//set to omit redundant nodes
+
```

Comment:
add comments. eg, what are the `String` stored in these data structures?

Reply:
Done